internal/scaler/awsec2asg/aws_ec2_asg.go

```diff
@@ -15,8 +15,10 @@ import (
 	"github.com/scriptnull/waymond/internal/scaler"
 )
 
-const Type scaler.Type = "aws_ec2_asg"
-
+const (
+	Type scaler.Type = "aws_ec2_asg"
+	AWS_REGION = "us-east-1"
```

Comment:
We would ideally like to get this information from the [waymond configuration file](https://scriptnull.github.io/waymond/docs/getting-started/config) instead of hard coding to a particular region.

Defining a `Region` field in `Scaler` struct in this source file should give us the region value if somebody had mentioned it in the waymond configuration.
```
Region *string `koanf:"region"`
```